templates/home.php

```diff
@@ -19,8 +19,19 @@
 	),
 );
 ?>
-
 <div class="event-page-wrapper">
+	<div class="notice" style="padding: .5rem">
+		<?php
+		echo wp_kses(
+			sprintf(
+			// translators: %s is a link to a page about hosting events.
+				__( 'Do you want to host your own event? <a href="%s">Find more information here</a>.', 'gp-translation-events' ),
+				'https://make.wordpress.org/polyglots/2024/05/29/translation-events-inviting-gtes-to-create-and-manage-events/'
```

Comment:
I have added a section in the [GlotPress handbook](https://make.wordpress.org/polyglots/handbook/for-editors/translation-collaboration-team-building-tips/translation-events-kit/#tracking-the-event-with-translation-events), so we can update the post URL to a more stable URL.
```suggestion
				'https://make.wordpress.org/polyglots/handbook/for-editors/translation-collaboration-team-building-tips/translation-events-kit/#tracking-the-event-with-translation-events'
```